supernode/daemon/mgr/gc/gc_peer.go

```diff
@@ -46,18 +46,16 @@ func (gcm *Manager) gcPeers(ctx context.Context) {
 			continue
 		}
 
-		if !gcm.gcPeer(ctx, peerID) {
-			logrus.Warnf("gc peers: failed to gc peer peerID(%s): %v", peerID, err)
-			continue
-		}
+		gcm.gcPeer(ctx, peerID)
 		gcPeerCount++
 	}
 
+	gcm.metrics.peersGCCount.WithLabelValues().Inc()
```

Comment:
You still need to update this. Use Add(gcPeerCount) instead of just Inc(). I have said that this metric is used to calculate the total peers being gc. Instead of total times of calling gc function.